### ISSUE TITLE
Refactor "sortBy" usage in FleetStatus and ProgressBarMulti

### ISSUE THREAD

**Issue by momesgin** [rancher/dashboard#10577]
**Describe the bug**

`sortBy` helper doesn't mutate the collection it receives as an argument, but returns a new sorted collection. It's been used the wrong way in `FleetStatus.vue` and `ProgressBarMulti.vue`. We need to investigate the side-effects if there are any, and fix the way it's being used.

**Comment by codyrancher** [comment]
@momesgin I just checked the two locations and both appear to be correct to me:
https://github.com/rancher/dashboard/blob/master/shell/components/fleet/FleetStatus.vue#L106
https://github.com/rancher/dashboard/blob/master/shell/components/ProgressBarMulti.vue#L84

Do you mind if we close this? If not, could you elaborate some more on what the problem is.

**Comment by momesgin** [comment]
> [@momesgin](https://github.com/momesgin) I just checked the two locations and both appear to be correct to me: https://github.com/rancher/dashboard/blob/master/shell/components/fleet/FleetStatus.vue#L106 https://github.com/rancher/dashboard/blob/master/shell/components/ProgressBarMulti.vue#L84
> 
> Do you mind if we close this? If not, could you elaborate some more on what the problem is.

You're right, since in both cases we're accessing the first item returned by `sortBy` we're using it correctly. Let's close it
